src/System.Collections.NonGeneric/src/System/Collections/Comparer.cs

```diff
@@ -34,6 +43,36 @@ public Comparer(CultureInfo culture)
             _compareInfo = culture.CompareInfo;
         }
 
+#if netstandard17
```

Comment:
Sorry I thought this was the test code (I got confused). For the test code that is going to remain we should try to split, however for product code here I agree we should just remove it. Is there any reason you cannot do that as part of this, it should be as simply as removing the older build configurations.